agent/bench-scripts/postprocess/fio-postprocess

```diff
@@ -155,14 +157,28 @@ if ($fio_results{"client_stats"}) {
 			}
 		}
 	}
-} else {
-	for my $client (@{ $fio_results{"client_stats"} }) {
-		for my $param (keys $$client{"job options"} ) {
+} elsif ($fio_results{"jobs"}) {
+	for my $job (@{ $fio_results{"jobs"} }) {
+		for my $param (keys $$job{"job options"} ) {
 			if (not $param =~/^write_\w+_log$/) {
-				$benchmark_params{$param} = $$client{"job options"}{$param};
+				if ($param =~/^numjobs$/) {
+					$num_jobs = $benchmark_params{$param};
+				} elsif ($param =~/^filename$/) {
+					if ($job_num % 4 == 0) {
```

Comment:
What is the significance of 4 that you are using it to % here?

Reply:
thanks for catching this.  It was supposed to be $numjobs